docs/source/release/v6.3.0/muon.rst

```diff
@@ -5,77 +5,67 @@ Muon Changes
 .. contents:: Table of Contents
    :local:
 
-.. warning:: **Developers:** Sort changes under appropriate heading
-    putting new features at the top of the section, followed by
-    improvements, followed by bug fixes.
-
 
 Frequency Domain Analysis
 -------------------------
 
 New Features
 ############
 
-- Can now select the unit for the frequency spectra (MHz or Gauss), by picking either `Frequency` or `Field` respectfully.
-
-Improvements
-############
+- Users can now select the unit for the frequency spectra (MHz or Gauss) in plotting, by picking either ``Frequency`` or ``Field`` respectively after a transform has been calculated.
 
 Bugfixes
 ########
 
 - Fixed a bug that prevented the frequency spectra from being plotted when data was binned.
 
-Muon Analysis
--------------
-
-New Features
-############
-
 
 Muon Analysis and Frequency Domain Analysis
 -------------------------------------------
 
 New Features
 ############
 
-- Instead of plotting the confidence interval of a fit as an error bar, it is now represented by a shaded region.
+- **Instead of plotting the confidence interval of a fit as an error bar, it is now represented by a shaded region.**
+
+.. image::  ../../images/Muon_error_shading.png
+   :align: center
+   :height: 800px
+
 
 Improvements
 ############
 
-- The `alpha` values on grouping tab are now to six decimal places.
-- The numerical values in the `run info` box on the home tab are now rounded to either 4 significant figures or a whole number, whichever is more precise.
+- Changes have been made to improve the speed of Muon Analysis and Frequency Domain Analysis.
+- The :ref:`Results Tab <muon_results_tab-ref>` will now dispay a warning (red text and a tooltip) if the results table already exists.
 - The results table now produces errors for log values (when they are available).
```

Comment:
is it deliberate that above has a link for the results tab and this does not?

Reply:
Are the results table and the result tab the same thing? I wasn't sure.

Reply:
ah no, sorry I misread it (missed the le)